Makefile

```diff
@@ -6,13 +6,29 @@ SHELL = /usr/bin/env bash -o pipefail
 MKFILE_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
 PROJECT_PATH := $(patsubst %/,%,$(dir $(MKFILE_PATH)))
 
-# VERSION defines the project version for the bundle.
-# Update this value when you upgrade the version of your project.
-# To re-generate a bundle for another specific version without changing the standard setup, you can:
-# - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
-# - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
+# Check for dirty.
+define check_dirty
+	GIT_SHA=$$(git rev-parse HEAD) || { \
+		GIT_HASH=$${GIT_SHA:-NO_SHA}; \
+	}; \
+	if [ -z "$$GIT_HASH" ]; then \
+		GIT_DIRTY=$$(git diff --stat); \
+		if [ -n "$$GIT_DIRTY" ]; then \
+			DIRTY="true"; \
+		else \
+			DIRTY="false"; \
+		fi; \
+	fi; \
+	echo $$DIRTY
+endef
+
 VERSION ?= 0.0.0
 
+# ldflag variables
+VERSION_LIMITADOR_OPERATOR=0.10.0-dev # change this when version increases
```

Comment:
I would rather have a `version.go` like https://github.com/Kuadrant/kuadrantctl/blob/main/version/version.go

Reply:
That makes sense, no worries, will implement!